sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/display/DisplayData.java

```diff
@@ -762,28 +576,27 @@ private static InternalBuilder forRoot(HasDisplayData instance) {
 
     @Override
     public Builder include(HasDisplayData subComponent) {
-      checkNotNull(subComponent);
+      checkNotNull(subComponent, "Input subcomponent cannot be null");
```

Comment:
`subComponent argument cannot be null` (etc. throughout)

I worry that "input" sounds too much like a *model* level thing, rather than the Java argument to this function.